src/Components/Address/__tests__/utils.jest.tsx

```diff
@@ -0,0 +1,238 @@
+import { Address } from "Components/Address/AddressForm"
+import { getYupAddressSchema } from "Components/Address/utils"
+
+describe.skip("getYupAddressSchema (local phone validation)", () => {
```

Comment:
This and the file it is testing might be candidates for deletion/consolidation elsewhere.